thirdparty/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+find_package(yaml-cpp)
```

Comment:
If you make this `find_package(yaml-cpp CONFIG)` then it should find yaml-cpp::yaml-cpp which means your src/OpenLoco/CMakeLists.txt changes are not required.

Reply:
Do I need to do the same for span-lite as it was also throwing an issue when I didn't load this in /src/OpenLoco/CMakeLists.txt?

Reply:
Argh just looking at yaml-cpp code they are missing the namespace part. I'll send them a PR that adds it as its the preferred way to do things nowadays. (No idea actually why it currently works then)

Reply:
On second thoughts it looks like no PR's ever get merged there. Therefore best approach is probably to drop the namespace. Can you go through and remove `yaml-cpp::` from all its users and remove the ` add_library(yaml-cpp::yaml-cpp ALIAS yaml-cpp)`

Reply:
> On second thoughts it looks like no PR's ever get merged there.

Yet another reason to trade `yaml-cpp` for `rapidyaml` in the not too distant future.

Reply:
This is done.